### DISCUSSION TITLE
reference genome for direct rna seq data

### DISCUSSION THREAD

**Discussion by YantingHuang** [GoekeLab/xpore#84] [Q&A] [answered]
Hi,

Thanks for the great software! I have a question regarding the alignment step:

I saw from the demo you provided that the reference genome you used is Homo_sapiens.GRCh38.cdna.ncrna_wtChrIS_modified-directRNA-32G_mapont.mmi. Could you provide a url linked to this file?

Also, I would like to confirm that whether this is the indexed file for a reference genome fasta.

Thanks,
Yanting

**Comment by ploy-np** [comment, marked as answer]
Hi @YantingHuang, 

Now I just uploaded to our zenodo (https://zenodo.org/record/5225625). And here is the direct link to the fastA file https://zenodo.org/record/5225625/files/Homo_sapiens.GRCh38.cdna.ncrna_wtChrIs_modified.fa
